Apps/Examples/Examples/All Examples/Custom2DPuckExample.swift

```diff
@@ -4,27 +4,125 @@ import MapboxMaps
 @objc(Custom2DPuckExample)
 public class Custom2DPuckExample: UIViewController, ExampleProtocol {
 
-    internal let toggleAccuracyRadiusButton: UIButton = UIButton(frame: .zero)
     internal var mapView: MapView!
-    internal var showsAccuracyRing: Bool = false {
+    internal var puckConfiguration = Puck2DConfiguration.makeDefault(showBearing: true)
+    private var showsPuck: PuckVisibility = .isVisible {
```

Comment:
imo this naming fits our convention better. But please feel free to dismiss this suggestion :-D
```suggestion
    private var isPuckVisible: PuckVisibility = .isVisible {
```

Reply:
The `showsPuck` name sounds good for me.